### PR TITLE
GROOVY-8423: Do not adjust access modifiers for inner class table ent…

### PR DESCRIPTION
**Pull request by shils**
…ries

### CODE REVIEW

File: src/main/java/org/codehaus/groovy/classgen/AsmClassGenerator.java [hunk 1]
```diff
@@ -305,11 +305,6 @@ private void makeInnerClassEntry(ClassNode cn) {
         }
         int mods = adjustedClassModifiersForInnerClassTable(cn);
 
-
-        if (Modifier.isPrivate(mods)) {
```
Comment (shils) [inline src/main/java/org/codehaus/groovy/classgen/AsmClassGenerator.java]: I'm unsure about removing this block as the commit message suggests it was necessary for android support. Does anyone know the history behind this? @melix

**Comment by paulk-asert** [comment]
+1 after android support feedback received. I didn't work on that aspect.

**Comment by melix** [comment]
I'm sorry I was sick the last few days I missed the ping. I can clearly remember that without this change the class format wasn't correct for Android. But it's far away and I haven't worked on this for years now. I'm afraid I'm no help.

**Comment by shils** [comment]
Thanks for reviewing, and hope you're feeling better. I'll come back to this once I've done more in-depth testing.

**Pull request closed without merging**